### PR TITLE
update gitignore (inspired by swift-numerics)

### PR DESCRIPTION
**Pull request by JaapWijnen**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,20 +1,9 @@
-## User settings
-xcuserdata/
```
Comment (rexmas) [inline .gitignore:2]: Why not include this one?

**Review by rexmas — COMMENTED** [review]
.

File: .gitignore (hunk 1 above)
Comment (JaapWijnen) [reply .gitignore:2]: Ah I mistook that for a subfolder of .swiftpm I can add it back in

**Comment by philipturner** [comment]
This is similar to a `.gitignore` that I use for most Swift projects:

https://github.com/philipturner/swift-opencl/blob/7e8587809b75774208fd8adfdd495613eabac0ff/.gitignore

**Review by philipturner — APPROVED** [review]

**Comment by JaapWijnen** [comment]
Yeah I basically looked at swift-numerics, swift-nio and swift-algorithms
@philipturner good to go then?

**Comment by philipturner** [comment]
I say good to go, but just need validation from @rexmas - please merge if you approve.

**Review by rexmas — APPROVED** [review]
Good, thanks!

**Pull request merged**